docs/guide/robot_sbc/setup_jetson_nano.md

```diff
@@ -66,76 +136,364 @@ pip3 install -U gdown
+Work through the __Prepare for Setup__, __Writing Image to the microSD Card__,
+and __Setup and First Boot__ instructions, then return here.
+
+Once you're done with the setup, ssh into your vehicle. Use the terminal for
+Ubuntu or Mac. [Putty](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html)
+for windows.
+
+
+Remove Libre Office:
 
 ```bash
-cd ~/projects
+sudo apt-get remove --purge libreoffice*
+sudo apt-get clean
+sudo apt-get autoremove
+```
+
+And add a 6GB swap file:
+
+```bash
+git clone https://github.com/JetsonHacksNano/installSwapfile
+cd installSwapfile
+./installSwapfile.sh
+reboot 
+```
+
+#### Step 2b: Free up the serial port (optional. Only needed if you're using the Robohat MM1)
+
+```bash
+sudo usermod -aG dialout <your username>
+sudo systemctl disable nvgetty
+```
+
+
+#### Step 3b: Setup Python Environment
+
+The installation on JP 4.6.X is more tricky, because we need to use a newer
+version of tensorflow than what is packaged by NVidia for this OS version.
+Also, we need a compatible OpenCV version which needs to be built on the nano.
+
+* Step 1: Install mamba-forge
+
+Download and install mambaforge
+
+```bash
+wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-aarch64.sh
+chmod u+x ./Mambaforge-Linux-aarch64.sh
+bash ./Mambaforge-Linux-aarch64.sh
 ```
 
-* Get the latest donkeycar from Github.
-> Note: There are currently version updates happening on the `main` branch so you might rather want to checkout the latest stable release as explained below.
+* Step 2: Install git lfs
 
 ```bash
+sudo apt-get install git-lfs
+```
+
+* Step 3: Download Donkey Car
+
+Downloading Donkey Car and the Tensorflow wheel for jp46 and tf29
+
+```bash
+mkdir projects
+cd projects
 git clone https://github.com/autorope/donkeycar
+git-lfs clone https://github.com/autorope/jetson
 cd donkeycar
 git checkout main
-pip3 install -e .[nano]
 ```
 
-* Alternatively, to get the latest stable release
+* Step 4: Create the python env and install donkey
+
+Also install tensorflow in the last step
+
+```bash
+mamba env create -f install/envs/jetson46.yml
+conda activate donkey
+conda update pip
+pip install -e .[nano]
+pip install git+https://github.com/autorope/keras-vis.git
+pip install ../jetson/tensorflow-2.9.3-cp39-cp39-linux_aarch64.whl
+```
+
+* Step 5: Check the TF installation
+
+Run python and verify that tensorflow is version 2.9 and trt is version 8.2.1:
+
+```bash
+python
+>>> import tensorflow as tf
+>>> tf.__version__
+>>> from tensorflow.python.compiler.tensorrt import trt_convert as trt
+>>> trt._check_trt_version_compatibility()
+```
+
+* Step 6: Install OpenCV
+
+OpenCV with GStreamer support is required for the Nano camera. However, the
+OpenCV binaries on PYPI don't have that enabled. Hence, we need to build these
+ourselves.
+
+Install opencv 4.6 + Gstreamer following the Q-Engineering Opencv manual install
```

Comment:
FYI: Q-engineering have OpenCV 4.7 instructions in their repo: [OpenCV-4-7-0.sh](https://github.com/Qengineering/Install-OpenCV-Jetson-Nano/blob/main/OpenCV-4-7-0.sh) but not on their tutorial page. Is there a Donkey Car preference for 4.6.0?

Reply:
I don't know, tbh, because we haven't tried. We were happy to find a combination of OpenCV with G - Streamer that worked with our version of python and jetpack. If you can confirm that the 4.7 works as well, then we are happy to update the docs accordingly.